homeworks/09-chokidar/src/importer/index.ts

```diff
@@ -0,0 +1,28 @@
+import fs from "fs";
+import { EventEmitter } from "events";
+import { csvToJson } from "csvtojson-converter";
+
+export class Importer {
+    constructor(
+        private readonly eventEmitter: EventEmitter
+    ) {}
+
+    public async listen(): Promise<never> {
+        this.eventEmitter.on("dirwatcher:changed", (file: string) => {
+            console.log(`New file detected [${file}]:`);
+            console.log(this.importSync(file));
+        });
+
+        return new Promise(() => {});
+    }
+
+    public async import(path: string): Promise<{ [key: string]: string }> {
```

Comment:
i guess that `[key: string]: string` can be exported to either interface of type.